Wed May 11 09:55:54 EEST 2011  Sergei Trofimovich <slyfox@community.haskell.org>
  * Fix build against ghc-7
diff -rN -u old-hipmunkplayground/HipmunkPlayground.cabal new-hipmunkplayground/HipmunkPlayground.cabal
--- old-hipmunkplayground/HipmunkPlayground.cabal	2011-05-11 09:57:33.929892005 +0300
+++ new-hipmunkplayground/HipmunkPlayground.cabal	2011-05-11 09:57:33.930892005 +0300
@@ -37,7 +37,7 @@
 
 Executable HipmunkPlayground
   if flag(small_base)
-    Build-Depends: base >= 3 && < 5, containers >= 0.1 && < 0.4
+    Build-Depends: base >= 3 && < 5, containers >= 0.1 && < 0.5
   else
     Build-Depends: base >= 2 && < 3
   Build-Depends: Hipmunk >= 5.2 && < 5.3, transformers >= 0.2 && < 0.3,
